test/conformance/kuttl/validate/clusterpolicy/standard/enforce/configmap-context-lookup/03-assert.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kyverno.io/v1alpha2
+kind: BackgroundScanReport
```

Comment:
Wouldn't we want to either in addition to or replacement of this check for a BackgroundScanReport check if it got written into a PolicyReport?

Reply:
I found an example using BackgroundScanReport to check the result so used it. I can update to polr.

Reply:
We could check both.